src/components/ParsedHtml/style.css

```diff
@@ -0,0 +1,8 @@
+.parsed-link {
+    color: #7F99FB !important;
```

Comment:
It will fail on light or dark theme, depending on which one you developed!

Reply:
Don't we have some default colors for links which would already do the job, without this CSS file?

Or maybe another way of adding this styling directly in React components?

Reply:
Should be `#2D4EC9` on the light theme, because `#7F99FB` fails the a11y compliance

Reply:
I verified and saw no issue on light nor dark theme.
Also those .css properties were already present before this PR, I just moved them in a separate file.

Reply:
So, you confirm that these parsed links have the `#2D4EC9` color on light mode, and `#7F99FB` on dark mode?